charts/project-origin-wallet/templates/deployment-wallet.yaml

```diff
@@ -106,6 +106,8 @@ spec:
               value: {{ .Values.config.jwt.audience }}
             - name: jwt__allowAnyJwtToken
               value: {{ .Values.config.jwt.allowAnyJwtToken | quote }}
+            - name: jwt__enableScopeValidation
+              value: {{ .Values.config.jwt.enableScopeValidation | quote }}
```

Comment:
Regarding the feature flag. When the time comes, is it expected that this feature flag is enabled by default?

That way one would explicitly have to opt out of using scopes or authorization, by using the [AllowAnonymous] annotation, no?

Reply:
I changed it to default enabled, but now users must be aware of breaking change!